### PR TITLE
git-pr: Remove page (en, es)

### PR DESCRIPTION
**Pull request by Amorymeltzer**
Added in #2674 (en) and #3727 (es) but sadly `git-pr` is not part of `git`, rather [`hub`](https://hub.github.com/hub.1.html#new-commands-provided-by-hub).  The en page for `hub` has an example for `pr` documented as of #3170, but there's no es page for hub (cc @ivanhercaz).

- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]

**Review by owenvoke — CHANGES_REQUESTED** [review]
I think that this shouldn't be applied. The `git-pr` page that I added was a reference to Git PR from [`git-extras`](https://github.com/tj/git-extras/blob/master/bin/git-pr) (which is not related to Hub). 🤔

I think what would make more sense is to add a link to these pages that reference:
https://github.com/tj/git-extras/blob/master/Commands.md#git-pr

i.e.
```md
> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-pr>.
```

**Comment by Amorymeltzer** [comment]
@owenvoke Indeed!  My mistake — I use them both and forget which does which.

Is it desirable for `tldr` to document `git-extras` commands?  If so, there are *plenty* to be added!  Is the header sufficient to clarify that these aren't built-in git commands?

**Comment by owenvoke** [comment]
In my opinion, yes! I think a lot of them are really useful to add to tldr pages. 👍🏻

**Comment by ivanhercaz** [comment]
> Added in #2674 (en) and #3727 (es) but sadly `git-pr` is not part of `git`, rather [`hub`](https://hub.github.com/hub.1.html#new-commands-provided-by-hub). The en page for `hub` has an example for `pr` documented as of #3170, but there's no es page for hub (cc @ivanhercaz).
> 
> * [x]  The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).

Excuse me a lot! I am really busy with work tasks and I have not been enough time to check all the GitHub notifications, and even more considering the activity of tldr-pages.

**Review by sbrl — CHANGES_REQUESTED** [review]
Just because they aren't core git subcommands, doesn't mean we shouldn't document them!

tldr-pages has a limitation that the name of a page  must exactly match the command being documented, so as log as you can access the `pr` extension git command by typing `git pr {{arguments_here}}`, we should keep it as-is.

**Comment by Amorymeltzer** [comment]
Looks like a consensus to me!  Closing this then; I've opened #4838 to do @owenvoke's [suggestion](https://github.com/tldr-pages/tldr/pull/4472#pullrequestreview-501655737)
